apps/api/src/app/import/import.service.ts

```diff
@@ -266,21 +266,18 @@ export class ImportService {
 
     const activities: Activity[] = [];
 
-    for (let [
-      index,
-      {
-        accountId,
-        comment,
-        currency,
-        date,
-        error,
-        fee,
-        quantity,
-        SymbolProfile,
-        type,
-        unitPrice
-      }
-    ] of activitiesExtendedWithErrors.entries()) {
+    for (const [index, activity] of activitiesExtendedWithErrors.entries()) {
+      const accountId = activity.accountId;
+      const comment = activity.comment;
+      const currency = activity.currency;
+      const date = activity.date;
+      const error = activity.error;
+      const quantity = activity.quantity;
+      const SymbolProfile = activity.SymbolProfile;
+      const type = activity.type;
+      let fee = activity.fee;
```

Comment:
I would move up even if it is with `let`

Reply:
You mean declare `fee` and `unitPrice` before the const declarations? so sort let before const?

Reply:
```
const accountId = activity.accountId;
const comment = activity.comment;
const currency = activity.currency;
const date = activity.date;
const error = activity.error;
let fee = activity.fee;
const SymbolProfile = activity.SymbolProfile;
const type = activity.type;
let unitPrice = activity.unitPrice;
```

Reply:
I see, now i understand :)